### ISSUE TITLE
"Done" option is not accessible for keyboard navigation

### ISSUE THREAD

**Issue by Fearinn** [LukeberryPi/phived#32]
The "done" option is made visible on hover, but nothing happens if the user just focus the input (with no hover). In addition, the option is a span, unfocusable and with no semantic role. To fix this and improve accessibility, we could:

- Make "done" visible on container focus;
- Implement "done" as a button instead of span.

This might be not the best solution, since I've not taken possible conflicts into account. Please correct me if I've missed something.

**Comment by LukeberryPi** [comment]
hey fearinn! a few thoughts on this:

i initially thought that showing the done button on container focus would be a good idea, but by using the app myself i found it bothering me more than helping. but i agree with you that a task must be able to be completed via keyboard only, without the obligation of a mouse.

which is why we implemented [ctrl + enter to complete a task](https://github.com/LukeberryPi/phived/issues/1). this and other shortcuts (such as [enter/shift + enter to move through tasks](https://github.com/LukeberryPi/phived/pull/5)) will be disclosed to the end user on the help menu, that is yet to be developed.

furthermore, there is a PR to be sent by @Allyhere that addreses the "done?" span being selectable via tab. if i see fit, i may send in the span => button change myself.

thanks for the issue!
